pom.xml

```diff
@@ -170,17 +170,18 @@
         <xalan.bundle.version>2.7.2_3</xalan.bundle.version>
         <!--END versions that MUST exactly match the versions used by the CXF and Camel features-->
         <!-- Artemis dependencies -->
-        <artemis.version>2.1.0</artemis.version>
+        <artemis.version>2.4.0</artemis.version>
         <!--START Camel and Artemis features should use the same versions for these bundles-->
-        <proton.version>0.19.0</proton.version>
```

Comment:
According to the comment above, the proton and qpid-jms versions used here should be kept in sync with the versions used in our Camel features.

Reply:
Does Camel have it's own version declared for this? Or are you saying I may need to upgrade Camel to a version that uses the same Proton version as Artemis?

Reply:
We maintain separate version properties for these dependencies that our Camel features use (in the platform-app pom). No you shouldn't need to upgrade camel itself, just these specific dependencies.

Reply:
It looks like the values for the camel versions correspond to the versions declared here.